analyzers/src/SonarAnalyzer.Common/Helpers/AspNetMvcHelper.cs

```diff
@@ -67,6 +67,7 @@ public static class AspNetMvcHelper
 
         public static bool ReferencesControllers(this Compilation compilation) =>
             compilation.GetTypeByMetadataName(KnownType.System_Web_Mvc_Controller) is not null
-            || compilation.GetTypeByMetadataName(KnownType.Microsoft_AspNetCore_Mvc_Controller) is not null;
+            || compilation.GetTypeByMetadataName(KnownType.Microsoft_AspNetCore_Mvc_Controller) is not null
+            || compilation.GetTypeByMetadataName(KnownType.Microsoft_AspNetCore_Mvc_ControllerBase) is not null;
```

Comment:
These two are in a different assembly, and it is possible to only reference one of them.